eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jwt/KeystoreProperties.java

```diff
@@ -0,0 +1,47 @@
+package uk.ac.ucl.eidp.auth.jwt;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Contains properties required to access the key store.
+ * @author rebmdgu
+ */
+public class KeystoreProperties {
+  
+  private final Properties properties = new Properties();
+  private final String propertiesPath = "META-INF/eidpauth.properties";
```

Comment:
\[FindBugs\] INFO: SS: Unread field: uk\.ac\.ucl\.eidp\.auth\.jwt\.KeystoreProperties\.propertiesPath; should this field be static?